octoprint_slic3r/profile.py

```diff
@@ -181,10 +182,9 @@ class SeamPositions(object):
     solid_infill_extruder = 1,
     support_material_contact_distance = 0.2,
     use_volumetric_e = 0,  				# 0 (unchecked) or 1 (checked)
-    threads = 2,
+    threads = min(multiprocessing.cpu_count() + 1, 16),
```

Comment:
Well, I just tried, if it makes any difference with values 5 and 2 on RPi2, and it seems there is little to none performance gain, probably there is not much to do in parallel. I'll revert it to 2 threds.